### DISCUSSION TITLE
Dúvida Curso Aplicando Orientação a Objetos em Projetos Reais com C# 11 e .NET 7

### DISCUSSION THREAD

**Discussion by andregr15** [balta-io/forum#554] [Tira Dúvidas] [answered]
Nos testes que esperam uma exceção, foi usado um try catch.

Existe algum benefício sobre o método Assert.ThrowsException<>?

**Comment by andrebaltieri** [comment, marked as answer]
@andregr15 não existe, é só uma abordagem diferente.
Talvez com ThrowsException fique melhor 🚀
